app/src/app/[lng]/[inventory]/InventoryResultTab/EmissionsForecast/GrowthRatesExplanationModalTable.tsx

```diff
@@ -34,20 +40,25 @@ export const GrowthRatesExplanationModalTable = ({
           </Tr>
         </Thead>
         <Tbody>
-          {SECTORS.map((sector) => (
-            <Tr key={sector.name}>
-              <Td>{t(sector.name + "-short")}</Td>
-              {Object.keys(growthRates)
-                .slice(0, 4)
-                .map((year) => (
-                  <Td key={year}>
-                    {growthRates[year][sector.referenceNumber]}
-                  </Td>
-                ))}
-              <Td>{growthRates["2030"][sector.referenceNumber]}</Td>
-              <Td>{growthRates["2050"][sector.referenceNumber]}</Td>
-            </Tr>
-          ))}
+          {[
+            ...SECTORS.slice(0, 4),
```

Comment:
I think we store if the sectors are BASIC or BASIC+ on them now, so there might be better ways to filter them than just the index.

This doesn't necessarily apply to the SECTORS constant, but could be added here too for more flexibility.

Not explicitly necessary for this PR though.

Reply:
we're not separating basic from basic+, only treats sector V differently because it uses the subsectors